### PR TITLE
DDF-1920: Updated karaf commands in documentation to karaf v.4 versions.

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?
With upgrade to karaf 4, many console commands changed names. These have been updated to reflect new names.
#### Who is reviewing it?
@pklinef @coyotesqrl @stustison @mcalcote @brianfelix @vinamartin @gordocanchola @clockard @ 
#### How should this be tested?
Compare to this table at http://karaf.apache.org/manual/latest-3.0.x/update-notes.html
#### What are the relevant tickets?
DDF-1920
#### Screenshots (if appropriate)
n/a
#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by ricklarsen** [comment]
`03-Mar-2016 12:01:03	[INFO] BUILD SUCCESS`

**Comment by vinamartin** [comment]
LGTM :+1:

File: catalog/spatial/docs/src/main/resources/_contents/mancontents.adoc [hunk 1]
```diff
@@ -38,42 +37,42 @@ These Installation steps are the same whether {branding} was installed from a di
...
-features:install <Application's main feature>
+feature:repo-add <Application's feature repository URL>
+feature:install <Application's main feature>
 ----
 
 .Example
 ----
-ddf@local>features:addurl mvn:ddf.catalog/catalog-app/2.3.0/xml/features
-ddf@local>features:install catalog-app
+ddf@local>feature:repo-add mvn:ddf.catalog/catalog-app/2.3.0/xml/features
```
Comment (coyotesqrl) [inline catalog/spatial/docs/src/main/resources/_contents/mancontents.adoc]: Should we maybe push the sample version number up to more closely reflect the present?

**Comment by pklinef** [comment]
:+1:

File: catalog/spatial/docs/src/main/resources/_contents/mancontents.adoc (hunk 1 above)
Comment (ricklarsen) [inline catalog/spatial/docs/src/main/resources/_contents/mancontents.adoc]: Not a bad idea. I can put in a property to stay in sync with the ${project.version}.

**Comment by coyotesqrl** [comment]
LGTM

**Comment by roelens8** [comment]
:+1:

File: pom.xml [hunk 2]
```diff
@@ -94,7 +94,7 @@
         <ddf-branding>DDF</ddf-branding>
         <ddf-branding-lowercase>ddf</ddf-branding-lowercase>
         <ddf-branding-expanded>Distributed Data Framework</ddf-branding-expanded>
-        <ddf-version>${project.version}</ddf-version>
+        <ddf.version>${project.version}</ddf.version>
```
Comment (clockard) [inline pom.xml]: Did you mean to change this to ddf.version instead of ddf-version. I think ddf-version is correct.

Comment (ricklarsen) [inline pom.xml]: I think either would work. It was just inconsistent.

**Pull request merged**